extensions/amp-analytics/0.1/transport.js

```diff
@@ -118,13 +152,208 @@ export class Transport {
     xhr.send('');
     return true;
   }
+
+  /**
+   * If iframe is specified in config/transport, check whether third-party
+   * iframe already exists, and if not, create it.
+   * @param {!Window} win The window element
+   * @param {!string} frameUrl  The URL of the cross-domain iframe
+   * @param {function(!string,
+   *   !../../../src/3p-analytics-common.AmpAnalytics3pResponse)=}
+   *   opt_processResponse An optional function to receive any response
+   *   messages back from the cross-domain iframe
+   */
+  processCrossDomainIframe(win, frameUrl, opt_processResponse) {
+    let frameData;
+    let shouldAddToDOM = false;
+    if (Transport.hasCrossDomainIframe(frameUrl)) {
+      frameData = Transport.getFrameData(frameUrl);
+      ++(frameData.usageCount);
+    } else {
+      frameData = this.createCrossDomainIframe(win, frameUrl);
+      shouldAddToDOM = true;
+    }
+    dev().assert(frameData, 'Trying to use non-existent frame');
+    if (opt_processResponse) {
+      Transport.responseProcessors_[this.id_] = opt_processResponse;
+    }
+    if (shouldAddToDOM) {
+      win.document.body.appendChild(frameData.frame);
```

Comment:
There was (see long comment in next function), but it is now obsolete. Good catch.